### PR TITLE
Compute also isScala211 only once

### PR DESCRIPTION
**Pull request by som-snytt**
Retronym also did this follow-up, which also
cleans up former cachings.

### CODE REVIEW

File: src/compiler/scala/tools/nsc/settings/ScalaSettings.scala [hunk 1]
```diff
@@ -83,7 +83,8 @@ trait ScalaSettings extends AbsScalaSettings
    * The previous "-source" option is intended to be used mainly
    * though this helper.
    */
-  def isScala211: Boolean = source.value >= ScalaVersion("2.11.0")
+  private[this] val version211 = ScalaVersion("2.11.0")
+  def isScala211: Boolean = source.value >= version211
```
Comment (janekdb) [inline src/compiler/scala/tools/nsc/settings/ScalaSettings.scala:87]: Off topic: `isGteqScala211` is a more accurate name!

Comment (som-snytt) [reply src/compiler/scala/tools/nsc/settings/ScalaSettings.scala:87]: On topic! Maybe `wasScala211`.

Comment (dwijnand) [reply src/compiler/scala/tools/nsc/settings/ScalaSettings.scala:87]: I've defined these before in build files as `isScala211Plus` (i.e `2.11+`).

Comment (janekdb) [reply src/compiler/scala/tools/nsc/settings/ScalaSettings.scala:87]: `atLeastScala211` is accurate and mildly more abstract than `gteqScala211`.

**Review by retronym — APPROVED** [review]
Thanks for following this up, LGTM.

**Comment by som-snytt** [comment]
How do they score this in the bug derby?

**Comment by retronym** [comment]
A solid 7.0, accounting for the degree of difficulty in dodging the question of `{is,was,isAtLeast,remindsOneOf}Scala211`

**Comment by som-snytt** [comment]
I meant in sports like basketball and hockey where they have "assists" or maybe baseball where a relief pitcher can get a "save". We need a scorecard. Or a "box score". FOSS can get complicated, and that's even modulo the licenses.

**Pull request merged**